### PR TITLE
NIAD-352: Map ITK Report -> FHIR Condition

### PR DESCRIPTION
**Pull request by Broom94**
Condition mapping complete with unit test and int test refactor, condition now added to bundle.

### CODE REVIEW

**Comment by Broom94** [comment]
Please note new code style schema add to project. I have reformatted all files I have edited for this PR incase you are wondering about changes to whitespaces etc. Thanks :)

**Comment by Broom94** [comment]
Should we merge NIAD-297 first to minimise potential conflicts as so many files have changed in that PR? @dawidskainos

**Comment by dawidskainos** [comment]
If you already used new formatter then I'll be able to resolve 297 conflicts easily. I don't want to block any stories because of formatting.

File: service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/ConditionMapper.java [hunk 1]
```diff
@@ -1,30 +1,47 @@
...
 
         condition.setIdElement(newRandomUuid());
 
-        condition
-                .setClinicalStatus(Condition.ConditionClinicalStatus.NULL)
-                .setVerificationStatus(Condition.ConditionVerificationStatus.UNKNOWN);
+        Date assertedDate = DateUtil.parse(itkEncounter.getEffectiveTime().getValue());
 
-        if (encounter.getText() != null) {
+        condition
```
Comment (dawidskainos) [inline service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/ConditionMapper.java:31]: I don't think setting targets is required here.

**Pull request merged**